src/NUnitFramework/mock-assembly/mock-nunit-assembly-4.5.csproj

```diff
@@ -39,6 +39,8 @@
     <DebugType>full</DebugType>
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>false</Prefer32Bit>
+    <Externalconsole>true</Externalconsole>
+    <Commandlineparameters>--test:NUnit.Tests.Assemblies.MockTestFixture</Commandlineparameters>
```

Comment:
Yes. Did the first commit from Linux. It's annoying how each IDE - even each VS version - has to put in it's own stuff. OTOH, I like how MonoDevelop saves the debug options in the project rather than the user file.